vancouver-drsm.csl

```diff
@@ -0,0 +1,466 @@
+<?xml version="1.0" encoding="utf-8"?>
+<style class="in-text" version="1.0" xmlns="http://purl.org/net/xbiblio/csl" demote-non-dropping-particle="sort-only" page-range-format="minimal" default-locale="fr-FR">
+  <info>
+    <title>Vancouver DRSM (French)</title>
+    <id>http://www.zotero.org/styles/vancouver-drsm</id>
+    <link rel="self" href="http://www.zotero.org/styles/vancouver-drsm"/>
+    <link href="http://www.zotero.org/styles/vancouver-fr-ca" rel="template"/>
```

Comment:
Also please add a "documentation" link to https://github.com/citation-style-language/styles/pull/2191, so it's clear where we can find more information about the style format. See also https://github.com/citation-style-language/styles/wiki/Style-Requirements#10---documentation-link